custom/src/main/java/co/elastic/otel/dynamicconfig/CentralConfig.java

```diff
@@ -95,7 +92,7 @@ private static String getServiceName(ConfigProperties properties) {
         return serviceName;
       }
     }
-    return "unknown_service"; // Specified default
+    return "unknown_service:java"; // Specified default
```

Comment:
Do we have a link to the specification here ? If yes, then maybe adding it as a comment would be helpful for future reference.